#### In murder trials in 20 Florida counties during 1976 and 1977, the death penalty was given in 19 out of 151 cases in which a white killed a white, in 0 out of 9 cases in which a white killed a black, in 11 out of 63 cases in which a black killed a white, and in 6 out of 103 cases in which a black killed a black (M. Radelet, Am. Sociol. Rev., 46: 918–927, 1981).

a. Exhibit the data as a three-way contingency table.

Z= Race of Victim 
X: Race of Defendant
Y= Death Penalty Verdict


In [2]:
Three_Way_Contingency_Table<-array(data=c(19,0,11,6,132,9,52,97), dim= c(4,2),
      dimnames=list(Victim.Race=c("White.White","White.Black","Black.White","Black.Black"),
                  Death.Penalty=c("Yes","No")))

Three_Way_Contingency_Table

#White.White = Victim (White), Defendant (White)
#White.Black = Victim (White), Defendant (Black)
#Black.White = Victim (Black), Defendant (White)
#Black.Black= Victim (Black), Defendant (Black)

Death.Penalty
Victim.Race   Yes No 
  White.White 19  132
  White.Black  0    9
  Black.White 11   52
  Black.Black  6   97

####  b. Construct the partial tables needed to study the conditional association between defendant’s race and the death penalty verdict. Find and interpret the sample conditional odds ratios, adding 0.5 to each cell to reduce the impact of the 0 cell count.

In [3]:
#Victim: White
dt_1 = data.frame("Victim" = c("White"),1,2,3,4, 
                    "Defendant.Race" = c("White", "Black"),2, 
                    "Death.Penalty" = c("Yes","No"))
 
PartialTable_1<-ftable(dt_1$Victim, dt_1$Defendant.Race, dt_1$Death.Penalty)
PartialTable_1[1,1] =132
PartialTable_1[1,2]=19
PartialTable_1[2,1]=9
PartialTable_1[2,2]=0

PartialTable_1

round(((19.5)*(9.5))/(132.5*0.5),2) #Adding .5 to each cell for 1st partial table
round((132.5*0.5)/(19.5*9.5),2)

              No Yes
                    
White Black  132  19
      White    9   0

[1] 2.8

[1] 0.36

The sample conditional odds ratio for partial table 1 is 2.80
Intrepretation: The sample odds for White defendants receiving the death penalty given the victim is White is 2.80 times greater than the odds for Black defendants.

In [4]:
#Victim: Black
dt_2 = data.frame("Victim" = c("Black"),1,2,3,4, 
                    "Defendant.Race" = c("White", "Black"),2, 
                    "Death.Penalty" = c("Yes","No"))

PartialTable_2<-ftable(dt_2$Victim, dt_2$Defendant.Race, dt_2$Death.Penalty)
PartialTable_2[1,1]= 52
PartialTable_2[1,2]=11
PartialTable_2[2,1]=97
PartialTable_2[2,2]=6
PartialTable_2

round((11*97)/(6*52),2)

             No Yes
                   
Black Black  52  11
      White  97   6

[1] 3.42

The sample conditional odds ratio for partial table 2 is 3.42

Intrepretation: The sample odds for White defendants receiving the death penalty, given the victim is Black, is 3.42 times greater than the odds for White defendants.

#### c. Compute and interpret the sample marginal odds ratio between defendant’s race and the death penalty verdict. Do these data exhibit Simpson’s paradox? Explain.

In [5]:
Marginal_Table<-array(data=c(30,6,184,106), dim= c(2,2),
      dimnames=list(Defendant.Race=c("White","Black"),
                  Death.Penalty=c("Yes","No")))
Marginal_Table
(30*106)/(6*184)
(6*184)/(30*106)

Death.Penalty
Defendant.Race Yes No 
         White 30  184
         Black  6  106

[1] 2.880435

[1] 0.3471698

The sample marginal odds ratio between defendant's race and the death penalty verdict shows that the odds of a White defendant receiving the death penalty is 2.88 times greater than the odds of a Black defendant.

These data exhibit Simpson's paradox because without accounting for the victim's race (confounding variable), it shows that there is greater tendency for White defendants to receive the death penalty. This is reflected in the sample marginal odds ratio. We can see that for the conditional association for table 1 the odds for White defendant is lower i.e. OR = 2.80



